src/domain/session/room/RoomViewModel.js

```diff
@@ -365,3 +388,13 @@ class ArchivedViewModel extends ViewModel {
         return "archived";
     }
 }
+
+class LowerPowerLevelViewModel extends ViewModel {
+    get description() {
+        return this.i18n`You do not have the powerlevel necessary to send messages`;
+    }
+
+    get kind() {
+        return "low-powerlevel";
```

Comment:
why not use the same kind (e.g. "disabled") for both `ArchivedViewModel` and this vm? the `kind` property is usually signalling how a vm should be rendered, so makes sense to use the same value if they are rendered in the same way.